docker-compose.deploy.volumes-placement.yml

```diff
@@ -7,7 +7,7 @@ services:
       placement:
         constraints:
           - node.role == manager
-          - node.labels.${STACK_NAME}.app-db-data == true
+          - node.labels.app-db-data == true
```

Comment:
This sets the label for the database volume to the one set in the documentation. I've already done this in the swarm.